builder/bundles.go

```diff
@@ -833,13 +851,13 @@ func installSpecialFilesToFull(b *Builder, packagerCmd []string, set *bundleSet,
 	}
 
 	// special handling for os-core
-	fmt.Println("... building special os-core content")
+	log.Info(log.Mixer, "... building special os-core content")
 	if err := buildOsCore(b, packagerCmd, fullDir, version); err != nil {
 		return err
 	}
 
 	// special handling for update bundle
-	fmt.Printf("... Adding swupd default values to %s bundle\n", b.Config.Swupd.Bundle)
+	log.Info(log.Mixer, "... Adding swupd default values to %s bundle", b.Config.Swupd.Bundle)
```

Comment:
Have this as `log.Info(log.Mixer, "Adding swupd default values to %s bundle", b.Config.Swupd.Bundle)`